### DISCUSSION TITLE
Can't locate my JSON file.

### DISCUSSION THREAD

**Discussion by bellsofaba** [EddieHubCommunity/LinkFree#9421] [Q&A] [answered]
So, apparently, LinkFree is now Biodrop. Way to go!

For some reason, I can't find my file json file. 

I sent a PR to add it about 3 months ago and it was successfully merged. 

Now I am trying to make some edits. 

Any help would be greatly appreciated.

**Comment by sandramsc** [comment, marked as answer]
Hi, the following might help:

1. Rename your forked repo from `LinkFree` to `BioDrop`
2. Sync fork to update branch inorder to merge the latest changes from the upstream repository into this branch.

**Reply by bellsofaba** [reply]
Thanks for the quick response @sandramsc 

I will have a go at it.

**Reply by eddiejaoude** [reply]
yep, make sure your fork is up to date 👍
